app/src/services/fetchArticles/fetchArticles.ts

```diff
@@ -0,0 +1,16 @@
+import {ArticlesResponse} from 'types/articles';
+
+const fetchArticles = async (
```

Comment:
I recommend considering handling various error cases, such as network errors, JSON errors, and handling the situation when the request is canceled (using Abort Controller, it is worth adding AbortError exception handling). Additionally, we can consider setting a timeout for the request to avoid application crashes.**